Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/AppCmdOnTargetMachines.ps1

```diff
@@ -192,6 +193,11 @@ function Add-SslCert
         return
     }
 
+    if($ipAddress -eq "All Unassigned"){
```

Comment:
addressed in commit [c26e546](https://github.com/Microsoft/vsts-rm-extensions/pull/159/commits/c26e5461667d1de2cfcd8d567a4b91b433341fd9)